### ISSUE TITLE
List of currently not (fully) working actions in Jellycuts 1.4

### ISSUE THREAD

**Issue by ghost** [ActuallyZach/Jellycuts-Issues#122]
# What action is the issue?
I made a list of currently not supported/working actions under iPadOS 16.1 with Jellycuts 1.4

## Describe the bug
All these actions below print an error. But whats strange: It also says that the "comment" action does not work, where it does in another shortcut. I also tried randomly filling out the variables to see if Jellycuts is then able to handle it, with no success...

## Steps To Reproduce
`import Shortcuts
#Color: grape, #Icon: shortcuts

//Unable to get shortcuts action is.workflow.actions.addnewevent
//Unable to get shortcuts action is.workflow.actions.addnewreminder
//Unable to get shortcuts action is.workflow.actions.converttimezone
//Unable to get shortcuts action is.workflow.actions.setters.calendarevents
//Unable to get shortcuts action is.workflow.actions.setters.reminders
//Unable to get shortcuts action is.workflow.actions.addnewcontact
//Unable to get shortcuts action is.workflow.actions.contacts
//Unable to get shortcuts action is.workflow.actions.setters.contacts
//Unable to get shortcuts action is.workflow.actions.email
//Unable to get shortcuts action is.workflow.actions.phonenumber
//Unable to get shortcuts action is.workflow.actions.venmo.request
//Unable to get shortcuts action is.workflow.actions.venmo.pay
//Unable to get shortcuts action com.apple.iBooks.openin
//Unable to get shortcuts action is.workflow.actions.dropbox.appendfile
//Unable to get shortcuts action is.workflow.actions.evernote.append
//Unable to get shortcuts action is.workflow.actions.evernote.new
//Unable to get shortcuts action is.workflow.actions.evernote.delete
//Unable to get shortcuts action is.workflow.actions.folder
//Unable to get shortcuts action is.workflow.actions.dropbox.open
//Unable to get shortcuts action is.workflow.actions.evernote.getlink
//Unable to get shortcuts action is.workflow.actions.evernote.get
//Unable to get shortcuts action is.workflow.actions.compresspdf
//Unable to get shortcuts action is.workflow.actions.filter.health.quantity
//Unable to get shortcuts action is.workflow.actions.health.quantity.log
//Unable to get shortcuts action is.workflow.actions.homeaccessory
//Unable to get shortcuts action is.workflow.actions.gethomeaccessorystate
//Unable to get shortcuts action is.workflow.actions.searchlocalbusinesses
//Unable to get shortcuts action is.workflow.actions.getdistance
//Unable to get shortcuts action is.workflow.actions.gethalfwaypoint
//Unable to get shortcuts action is.workflow.actions.getmapslink
//Unable to get shortcuts action is.workflow.actions.gettraveltime
//Unable to get shortcuts action is.workflow.actions.location
//Unable to get shortcuts action is.workflow.actions.getdirections
//Unable to get shortcuts action is.workflow.actions.searchmaps
//Unable to get shortcuts action is.workflow.actions.ride.requestride
//Unable to get shortcuts action is.workflow.actions.address
//Unable to get shortcuts action is.workflow.actions.image.removebackground
//Unable to get shortcuts action is.workflow.actions.listeningmode.set
//Unable to get shortcuts action is.workflow.actions.imgur.upload
//Unable to get shortcuts action is.workflow.actions.appendvariable
//Unable to get shortcuts action is.workflow.actions.comment
//Unable to get shortcuts action is.workflow.actions.measurement.convert
//Unable to get shortcuts action is.workflow.actions.dismisssiri
//Unable to get shortcuts action is.workflow.actions.dnd.getfocus
//Unable to get shortcuts action is.workflow.actions.getvariable
//Unable to get shortcuts action is.workflow.actions.returntohomescreen
//Unable to get shortcuts action is.workflow.actions.measurement.create
//Unable to get shortcuts action is.workflow.actions.personalhotspot.set
//Unable to get shortcuts action is.workflow.actions.setvariable
//Unable to get shortcuts action is.workflow.actions.tumblr.post
//Unable to get shortcuts action is.workflow.actions.wordpress.post
//Unable to get shortcuts action is.workflow.actions.sendemail
//Unable to get shortcuts action is.workflow.actions.sendmessage
//Unable to get shortcuts action is.workflow.actions.cloudapp.upload
//Unable to get shortcuts action is.workflow.actions.trello.add.card
//Unable to get shortcuts action is.workflow.actions.trello.add.board
//Unable to get shortcuts action is.workflow.actions.trello.add.list
//Unable to get shortcuts action is.workflow.actions.properties.trello
//Unable to get shortcuts action is.workflow.actions.trello.get
//Unable to get shortcuts action is.workflow.actions.instapaper.add
//Unable to get shortcuts action is.workflow.actions.pinboard.add
//Unable to get shortcuts action is.workflow.actions.pocket.add
//Unable to get shortcuts action is.workflow.actions.giphy
//Unable to get shortcuts action is.workflow.actions.instapaper.get
//Unable to get shortcuts action is.workflow.actions.pocket.get
//Unable to get shortcuts action is.workflow.actions.pinboard.get
`

### Link
https://www.icloud.com/shortcuts/5bf6a845a33e40878378a45b55d460b0

## Information
 - iPad Pro
 - OS: 16.1
 - Jellycuts Version 1.4

### Additional context
I did not test any 3rd party actions from Apps like Toolbox Pro, Actions, Data Jar, Jayson, Scriptable etc. All these actions are plain native Apple Actions.

**Comment by ActuallyTaylor** [comment]
This is an awesome list! I am currently working on fixing up the entire core of Jellycuts to make it much easier to support the new actions. Once I get this code upgrade working, I will get these actions into the app.

**Comment by ghost** [comment]
If it helps, I can also go through popular 3rd party apps like Toolbox Pro etc. to provide for them feedback too?! 

EDIT: I just tipped to get the Pro version to add 3rd party libraries like ToolBox Pro, awesome. 
When I import a shortcut with the `Add Contact` action from TBP I get this: 

> //Unable to get shortcuts action com.alexhay.ToolboxProForShortcuts.AddContactIntent

When I then try to compile with the library `import ToolboxPro` it stays as it is. 
Only if I modify the jellycut file with the additional library and the action described in your documentation and export it back to Shortcuts, I see that it is working, but it does not refresh/update the imported file, because I can only add the additional libraries like TBP afterwards.

Is there a way to let the app check if any 3rd party app was used and if thats true, add the corresponding library with i.e. `import ToolboxPro` automatically?

**Comment by ActuallyTaylor** [comment]
Yes I can definitely add the feature for library checking. Currently I am working on a full rewrite for the underlying systems of jellycuts to avoid all of these issues in the future.

**Comment by chrisjmendez** [comment]
This issue tracked action coverage gaps from Jellycuts 1.4. Action parsing/data coverage has been substantially updated in current versions and this historical aggregate list is no longer actionable as a single bug. Closing in favor of filing any remaining unsupported actions as focused issues.
